yeahcheese/resources/views/search.blade.php

```diff
@@ -6,6 +6,11 @@
     <h2>イベント閲覧</h2>
     <h1>認証キー入力</h1>
     <div>
+    @if ($errors->any())
+        @foreach ($errors->all() as $error)
+        <p>{{ $error }}</p>
+        @endforeach
```

Comment:
表示するエラー1個しかないから
```suggestion
        <p>{{ $errors->first() }}</p>
```
とかでも良さそう。
直すか直さないかは任せます。

Reply:
レビューありがとうございます！
search画面で表示するエラーは今後もこの一種類だけだと思うので、first()を使う形に修正します。